doc/changelog.rst

```diff
@@ -58,6 +58,9 @@ Bug Fixes
 - Skips conformers with no atoms in finding unique conformers
 - Corrects benchmark TS location for the Grubbs metathesis example, where the reactant complex is bound
 - Fixes possible zero distance constraint for a single atom
+- Fixes spin state definition for XTB calculations
```

Comment:
erm, this doesn't appear to be in this PR?

Reply:
Forgot to update the changelog from the last one 🤦, hence the change here